followthemoney/cli/ocds.py

```diff
@@ -10,6 +10,8 @@
 log = logging.getLogger(__name__)
 IDENTIFIERS = {
     'TRADE_REGISTER': 'registrationNumber',
+    'AU-ABN': 'registrationNumber',
+    'PY-PGN': 'classification',
```

Comment:
is there a source for these somewhere?

Reply:
http://org-id.guide/results?structure=all&coverage=all&sector=all

Reply:
Hm, I don't think we want to drag in all that complexity. Maybe let's just add a default to `IDENTIFIERS.get` for `registrationNumber`?